package.json

```diff
@@ -2,25 +2,29 @@
   "name": "ml-regression-simple-linear",
   "version": "2.0.3",
   "description": "Simple Linear Regression",
-  "main": "lib/index.js",
-  "module": "src/index.js",
-  "types": "regression-simple-linear.d.ts",
+  "main": "./lib/index.js",
+  "module": "./lib-esm/index.js",
+  "types": "./lib/index.d.ts",
   "sideEffects": false,
   "files": [
-    "regression-simple-linear.d.ts",
     "lib",
+    "lib-esm",
     "src"
   ],
   "scripts": {
-    "compile": "rollup -c",
+    "check-types": "tsc --noEmit",
+    "clean": "rimraf lib lib-esm",
     "eslint": "eslint src",
     "eslint-fix": "npm run eslint -- --fix",
-    "prepack": "npm run compile",
+    "prepack": "npm run tsc",
     "prettier": "prettier --check src",
     "prettier-write": "prettier --write src",
-    "test": "npm run test-coverage && npm run eslint && npm run prettier",
-    "test-only": "jest",
-    "test-coverage": "jest --coverage"
+    "test": "npm run test-coverage && npm run eslint && npm run prettier && npm run check-types",
+    "test-only": "vitest --run",
+    "test-coverage": "vitest --run --coverage",
```

Comment:
You can remove the `test-coverage` script and add the coverage option to `test-only`.